website/docs/spec/index.md

```diff
@@ -229,6 +268,18 @@ A Schema and Channel record MUST exist in the summary section for all channels r
 
 > Why? The typical use case for file readers using an index is fast random access to a specific message timestamp. Channel is a prerequisite for decoding Message record data. Without an easy-to-access copy of the Channel records, readers would need to search for Channel records from the start of the file, degrading random access read performance.
 
+### Secondary Chunk Index (op=0x12)
+
+A secondary Chunk Index record contains additional secondary index information on top of the corresponding Chunk Index record.
+
+| Bytes | Name                  | Type                  | Description                                                                                                                                                                                                                       |
+| ----- | --------------------- | --------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 2     | secondary_index_id    | uint16                | Secondary Index ID.                                                                                                                                                                                                               |
+| 8     | chunk_start_offset    | uint64                | Offset to the chunk record from the start of the file.                                                                                                                                                                            |
+| 8     | earliest_key          | Timestamp             | Earliest key in the chunk. Zero if the chunk contains no messages with this key.                                                                                                                                                  |
+| 8     | latest_key            | Timestamp             | Latest key in the chunk. Zero if the chunk contains no messages with this key.                                                                                                                                                    |
```

Comment:
using a zero timestamp as a sentinal is something we do elsewhere but not strictly correct, since 0 is a valid timestamp.